react/Viewer/index.jsx

```diff
@@ -72,6 +72,7 @@ export class Viewer extends Component {
   render() {
     const {
       currentFile,
+      currentURL,
```

Comment:
as said in the readme.md , let's add a check if currentURL not set and file encrypted 🙏